src/server/response.cpp

```diff
@@ -397,7 +397,7 @@ ContentResponse::contentDecorationAllowed() const
   if (m_raw) {
     return false;
   }
-  return (startsWith(m_mimeType, "text/html")
+  return ( (startsWith(m_mimeType, "text/html") || startsWith(m_mimeType, "application/xml"))
```

Comment:
Though this change addresses my earlier comment in an explicit way, it is actually not needed (in fact it enables decoration of XML content unless it is labeled with ";raw=true")